docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -709,6 +709,55 @@ Key interfaces
         :raises cryptography.exceptions.InvalidSignature: If the signature does
             not validate.
 
+    .. method:: recover_data_from_signature(signature, padding, algorithm)
+
+        .. versionadded:: 3.3
+
+        Recovers the signed data from the signature. The data contains the
```

Comment:
```suggestion
        Recovers the signed data from the signature. The data typically contains the
```

Reply:
This may make the line too long and require re-wrapping but it should say typically since just below we note cases where it may not be.